README.md

```diff
@@ -1798,4 +1798,11 @@ P.S that's Riziy and he help me study ;)
 ![Riziy kitten](assets/images/riziy-vcheryasova.jpg)
 
 # Hi everyone! :+1:
-[!cat](gif/kitty2.gif)
\ No newline at end of file
+[!cat](gif/kitty2.gif)
+
+---
+
+## Hello everyone, have a nice day and successful training!
+[Cat](https://undark.org/wp-content/uploads/2020/02/GettyImages-1199242002-1-scaled.jpg)
```

Comment:
You missed "!"  here [Cat]  
That"s why your picture is not displayed. Could you please fix it?